radis/default_radis.json

```
@@ -19,6 +19,7 @@
     "AUTO_UPDATE_DATABASE": false           # auto-convert 'pytables' files to 'vaex'
     "AUTO_UPDATE_SPEC": false               # auto-update saved Spectrum objects to latest format.
     "OLDEST_COMPATIBLE_VERSION": "0.9.1"    # automatically regenerate cache, files generated with versions anterior to this one,
+    "PREVENT_LARGE_DOWNLOAD_ABOVE_X_GB": 1  # prevent downloading databases larger than X GB,
```

I realize that this variable is not named correctly as it does *not* prevent a large download. It just warns the user. It should be renamed "
```suggestion
    "WARN_LARGE_DOWNLOAD_ABOVE_X_GB": 1  # Warn the user when downloading databases larger than X GB but does *not* stop the download,
```

---

agreed